src/components/AddFreakModal/AddFreakModal.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Modal from '../Modal/Modal';
+import EditFreakForm from '../EditFreakForm/EditFreakForm';
+import { FreakModelDefault } from '../../models/freaks';
+import AppButton from '../AppButton/AppButton';
+
+function AddFreakModal({ title, isOpen, onClose }) {
+  const [ freak, setFreak ] = useState(FreakModelDefault);
+
+  function handleSubmit(e) {
+    e.preventDefault();
+    console.log(freak);
+  }
+
+  return (
+    <Modal
+      title={ title }
+      isOpen={ isOpen }
+      headerContent={ <AppButton btnText="Reset" btnStyle="gray" /> }
+      footerContent={ <AppButton form="add-freak-form" btnText="Submit" btnType="submit" btnVariant="primary" btnStyle="py-2 px-5" /> }
```

Comment:
Rename these props to not have the btn prefix, as you do not need it.

btnText -> label
btnType -> type
.. etc